feature-flag/src/main/kotlin/com/linecorp/android/featureflag/model/FeatureFlagOption.kt

```diff
@@ -38,4 +38,8 @@ internal enum class FeatureFlagOption {
      * This option cannot be used with [OVERRIDABLE] option at the same time.
      */
     LITERALIZE,
+    /**
+     * An option to make a flag value deprecated.
```

Comment:
Please rephrase as follows

"""
An option to annotate a flag with \[Deprecated\].
"""

And update line 20 as "...visibility, mutability, and annotations."

Reply:
Done